m1.wack-a-mole/wackAmole.js

```diff
@@ -0,0 +1,144 @@
+
+let IntervalFunc;
+let randNums;
+let score = 0;
```

Comment:
A better practice would be to use JQuery `document.ready` scope for global variables (so they are encapsulated and cannot be accessed by `window.VARIABLE`